README.md

```diff
@@ -8,110 +8,89 @@
 ![CI Check Golang](https://github.com/intel/confidential-cloud-native-primitives/actions/workflows/pr-golang-check.yaml/badge.svg)
 ![CI Check Container](https://github.com/intel/confidential-cloud-native-primitives/actions/workflows/pr-container-check.yaml/badge.svg)
 ![CC Foundation Image Customize](https://github.com/intel/confidential-cloud-native-primitives/actions/workflows/image-rewriter.yaml/badge.svg)
+[![OpenSSF Best Practices](https://www.bestpractices.dev/projects/8325/badge)](https://www.bestpractices.dev/projects/8325)
 
 ## 1. Introduction
 
-Confidential Computing technology like Intel TDX provides isolated encryption runtime
+Confidential Computing technology like Intel® TDX provides an isolated encryption runtime
```

Comment:
I just noticed this should be  "Confidential Computer technologies"  Another way to say it would be "A Confidential Computing technology like" ..  I prefer the first